### PR TITLE
Enable driver for RTL8111H used in cm4 carrier boards

### PR DESCRIPTION
**Pull request by sspans**
The Realtek RTL8111h chip seems to be a popular choice for additional ethernet ports on CM4 carrier boards.
It would make sense to build the module by default and save users the hassle of building it themselves.

Carrier boards using this ethernet chip:

- https://www.dfrobot.com/product-2242.html
- https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/136
- https://blog.zakkemble.net/rpi-compiling-a-module-for-the-64-bit-kernel/

GitHub / GitLab issues discussing these boards / the driver:
- https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/114
- https://gitlab.alpinelinux.org/alpine/aports/-/issues/12662

### CODE REVIEW

File: arch/arm/configs/bcmrpi_defconfig [hunk 1]
```diff
@@ -497,6 +497,7 @@ CONFIG_NET_VRF=m
 CONFIG_ENC28J60=m
 CONFIG_QCA7000_SPI=m
 CONFIG_QCA7000_UART=m
+CONFIG_R8169=m
```
Comment (pelwell) [inline arch/arm/configs/bcmrpi_defconfig]: This is the wrong file - the configuration for 32-bit CM4 kernels comes from `arch/arm/configs/bcm2711_defconfig`.

Comment (sspans) [reply arch/arm/configs/bcmrpi_defconfig]: Thanks for the correction, I've updated the branch.

**Comment by geerlingguy** [comment]
Just wanted to note the 52Pi CM4 Router Board also uses this chipset: https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/337

**Comment by sspans** [comment]
Driver is in the unstable firmware, I hope it’ll hit the packaged kernels soon.

**Comment by sspans** [comment]
Driver is available in the most recent raspberrypi-firmware (1:1.20220120-1) package.

**Pull request merged**